public/js/products.js

```diff
@@ -0,0 +1,85 @@
+(function(item, cartCountElement) {

```

Comment:
Why you checking your public folder? It should have all build process generated files.